### ISSUE TITLE
Canal para comércio no Slack

### ISSUE THREAD

**Issue by giovannicruz97** [training-center/forum#37]
Olá, humanos! 👽 

O usuário @ lucasmacedo (no Slack) questionou se havia algum canal para a venda de livros e isso me fez pensar num canal para a nossa comunidade do Slack, o #barganha ou #taverna (ou qualquer um que decidirem 😆). Dessa maneira, podemos comercializar itens dentro da nossa comunidade (livros, notebooks, celulares e por aí vai), tudo dentro das leis claro! Sugestões são bem-vindas!

Valeu! 🤘

**Comment by lucassmacedo** [comment]
Top!!! Aguardando!

**Comment by manzoliric** [comment]
É um pouco complicado, porque nesse mundo de "rolo" venda e trocas, sempre pode dar algum problema e o TC não pode ser o intermediário disso...

Um exemplo, alguém vende um celular pelo TC e o mesmo da problema com o novo dono e começa aquela discussão... então acho que se for criar algo assim, o usuário tem que ter noção total que o TC não tem nada haver, um exemplo disso é o OLX, tem muitos anúncios, mas ele não tem garantia nenhuma.

**Comment by gionaufal** [comment]
Pensei nisso esses dias, também. Acho válido por ser uma comunidade com muita gente e é bem possível que o que eu quero comprar alguém queira vender, e vice-versa. Mas se isso for pra frente certamente deve ser levado em consideração o que o @richmanzoli disse, que o TC não tem nenhuma relação com as ofertas e ninguém da moderação deveria se responsabilizar por qualquer problema que possa ocorrer entre as partes.

**Comment by giovannicruz97** [comment]
Concordo com o que foi dito acima. Podemos criar um conjunto de regras para o livre comércio dos usuários e o TC esta longe de ser um OLX. A grande sacada é conectar as pessoas.

**Comment by adeonir** [comment]
Acho bacana essa ideia, mas deveria ficar aqui no github mesmo, onde as mensagens não são perdidas.
Sobre negociar notebooks, celulares e afins, sei não... mas livros técnicos super dahora.

**Comment by giovannicruz97** [comment]
Acredito que podemos ter essa área de comércio, pois as outras comunidades também possuem e funciona super bem. Não podemos pensar somente no lado negativo do ser humano

**Comment by woliveiras** [comment]
Galera, um dos objetivos do TC é compartilhar conhecimento pra mudar a vida das pessoas e a única maneira de mudar a vida **de quem realmente precisa** é disseminar o conhecimento de maneira **gratuita**. Afinal muita gente não consegue pagar um valor alto pelos livros/cursos da nossa área.

Os da Casa do Código são um ótimo exemplo de livros acessíveis, mas mesmo assim ainda temos pessoas que não podem dispor do valor. :disappointed: 

Seria melhor a doação do que a venda de livros no Training Center, pois estariamos ajudando alguém que não pode comprar.

Se for um canal para doações, poderia ser tudo feito dentro do TC, mas um canal de vendas não é legal ter lá.

Mas é claro que não podemos limitar vocês... Se for pra vender algo, poderia existir o canal, mas a venda não deveria ser feita no Slack. Deveria ser um canal só pra enviar link do Mercado Livre ou da OLX, por exemplo, pois daí tem a proteção do site e o TC não se responsabilizaria por isso.

Eu só gostaria que vocês estimulassem mais a doação de livros do que venda dos mesmos. :pray: 

Lembrem-se que alguém do outro lado do país pode estar passando por dificuldades financeiras e você mudaria a vida dela com essa atitude. :metal:

**Comment by lflimeira** [comment]
Assim como comentei no Slack, eu apoio totalmente o ponto de vista do @woliveiras. Se for para doação seria muito massa, mas para comércio eu n acho que o TC seja um lugar para isso 😆

**Comment by jaqueventurim** [comment]
Poh super concordo com a idéia de doações rs.

Se pensarmos o que o TC quer estimular poderíamos pensar nessa forma de doações pra firmar ainda mais nosso compromisso com quem as vezes não tem como dispor de grana pra acessar conhecimento né! Curti ^^

Vou até pensar em algum que eu possa doar rs.
